superset-frontend/src/SqlLab/components/TableElement.jsx

```diff
@@ -110,7 +110,7 @@ class TableElement extends React.PureComponent {
           />
         );
       }
-      let latest = Object.entries(table.partitions.latest).map(
+      let latest = Object.entries(table.partitions?.latest || []).map(
```

Comment:
This isn't necessary anymore given the issue being addressed in the backend, but is a tiny bit of defensive programming.